includes/wizards/audience/class-audience-content-gates.php

```diff
@@ -142,6 +142,22 @@ public function register_api_endpoints() {
 			]
 		);
 
+		register_rest_route(
+			NEWSPACK_API_NAMESPACE,
+			'/content-gate/priority',
+			[
+				'methods'             => 'POST',
+				'callback'            => [ $this, 'update_gate_priorities' ],
+				'permission_callback' => [ $this, 'api_permissions_check' ],
+				'sanitize_callback'   => [ $this, 'sanitize_gates' ],
+				'args'                => [
+					'gates' => [
+						'type' => 'array',
+					],
+				],
```

Comment:
We can handle this in another PR, but it would probably be best to just send the order values here instead of the entire gate payload.